Program.markdown

```diff
@@ -80,39 +92,48 @@ For at kommunikasjonen skal tjene formålet med å koble sammen individer, må i
 
 - Lovfeste tilgang til Internett. For at borgere skal kunne delta i samfunnet i dag må de ha tilgang til Internett.
 - Øke nettutbygging med spesiell fokus på fiberutbygging. Øke ambisjonsnivået mht. netthastigheter, og ta med både opp- og ned-hastigheter i kravene.
-- Reservere frekvensområder til fri benyttelse (eks. WiFi, «white spaces»).
+- Forby kvoter på bredbånd mobilt eller trådet.
```

Comment:
Kopiert riktig. Ikkje endre på det no.
Eg vil berre notere ned eit omformuleringsforslag her, som gjer at setninga blir klarare:
"Forby kvoter på bredbånd, både for mobilt og kablet."